setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: single
+title:  "Pipeline Permissions"
+sidebar:
+  nav: setup
+---
+
+{% include alpha version="1.9" %}
+
+{% include toc %}
+
+Pipeline permissions enable automatically triggered pipelines to modify
+resources in protected accounts and applications. They are an alternative
+to manually managing Fiat Service Accounts.
+
+Previously, a Spinnaker operator would have to first create a Fiat Service
```

Comment:
It's better not to place this is time, with words like "previously" (and below, you use "currently").

Instead, can you contrast pipeline permissions with the service-accounts way, but without making them old versus new?

Reply:
done.

Reply:
:+1: